<ui version="4.0" >
 <class>PathFinderClass</class>
 <widget class="QDialog" name="PathFinderClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Manual network creation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="0" column="0" colspan="4" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Available metabolites</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="margin" >
       <number>9</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="treeWidget" >
        <column>
         <property name="text" >
          <string>Compartments</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox" >
        <property name="mouseTracking" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Ignore compartments</string>
        </property>
        <property name="checked" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item rowspan="7" row="0" column="4" >
    <widget class="QGroupBox" name="groupBox_5" >
     <property name="title" >
      <string>User settings</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="margin" >
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Metabolite blacklist</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QTextEdit" name="textEdit_3" />
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_11" >
        <property name="text" >
         <string>User defined reaction properties</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit" />
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6" >
        <property name="maximumSize" >
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text" >
         <string>Apply changes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_7" >
     <property name="text" >
      <string>Organism</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3" >
    <widget class="QComboBox" name="comboBox_4" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="currentIndex" >
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4" >
    <widget class="QGroupBox" name="groupBox_4" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <property name="title" >
      <string>Transition condition</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Match if</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="comboBox" >
        <property name="currentIndex" >
         <number>0</number>
        </property>
        <item>
         <property name="text" >
          <string>any</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>transition is of type "main"</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>carbon transition occurs</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>label transition occurs</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_10" >
        <property name="text" >
         <string>If no match</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="comboBox_5" >
        <property name="currentIndex" >
         <number>1</number>
        </property>
        <item>
         <property name="text" >
          <string>take every path (of that reaction)</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>take no path (skip reaction)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="4" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Startmetabolite</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" colspan="2" >
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="label_9" >
        <property name="text" >
         <string>Compartment</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="QLineEdit" name="lineEdit" >
        <property name="text" >
         <string>citrate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2" >
       <widget class="QComboBox" name="comboBox_2" />
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Labels</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2" >
       <widget class="QLineEdit" name="lineEdit_5" >
        <property name="text" >
         <string>none</string>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QPushButton" name="pushButton_5" >
        <property name="text" >
         <string>Select</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Network depth</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <widget class="QLineEdit" name="lineEdit_3" >
     <property name="text" >
      <string>3</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" >
    <widget class="QCheckBox" name="checkBox_RankedGraph" >
     <property name="text" >
      <string>Ranked graph</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2" >
    <widget class="QPushButton" name="pushButton" >
     <property name="text" >
      <string> Create network</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" >
    <widget class="QPushButton" name="pushButton_4" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Show graph</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3" >
    <widget class="QCheckBox" name="checkBox_5" >
     <property name="text" >
      <string>Quit if Endmetabolite reached</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4" >
    <widget class="QGroupBox" name="groupBox_6" >
     <property name="title" >
      <string>Path Finder</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Endmetabolite</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="lineEdit_2" />
      </item>
      <item row="0" column="2" >
       <widget class="QComboBox" name="comboBox_3" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Max path length</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="lineEdit_4" >
        <property name="text" >
         <string>10</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QPushButton" name="pushButton_2" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Find paths</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2" >
       <widget class="QPushButton" name="pushButton_3" >
        <property name="text" >
         <string>Show path viewer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
